packages/auth/src/CartoProvider.ts

```diff
@@ -0,0 +1,54 @@
+import { OAuth2Provider } from '@salte-auth/salte-auth';
+
+interface CartoValidation extends OAuth2Provider.Validation {
+  user_info_url: string;
+}
+
+export class CartoProvider extends OAuth2Provider {
+  private _userInfo: string = this.get('user_info');
+
+  get name() {
+    return 'carto';
+  }
+
+  get login() {
+    return `${this.config.url}/authorize`;
+  }
+
+  get logout() {
+    return `${this.config.url}/logout`;
+  }
+
+  get userInfo() {
+    return this._userInfo;
+  }
+
+  public $validate(options: CartoValidation): void {
```

Comment:
I can see the override here, but... Do we really need to keep the dollar prefix here? Isn't it a bit strange for the final user?

Reply:
You should not call that directly imho. The only reason why it's public is because it is public on salte-auth :man_shrugging: